cs/unittest/RunTests.tt

```diff
@@ -31,8 +31,8 @@ var skipList = new[] { 13, 32, 39, 40, 41, 59, 60, 61, 66, 68, 90,
     220, // deamon mode shell script test not supported
```

Comment:
on line 27: you want to 260 to become 258 based on your changes on Runtests

Reply:
Thanks